nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -160,6 +160,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [openvscode-server](https://github.com/gitpod-io/openvscode-server), run VS Code on a remote machine with access through a modern web browser from any device, anywhere. Available as [services.openvscode-server](#opt-services.openvscode-server.enable).
 
+- [ios-webkit-debug-proxy](https://github.com/google/ios-webkit-debug-proxy), a DevTools proxy for iOS devices.
```

Comment:
This should be moved to 23.11 or be dropped since the package is probably not that note worthy

Reply:
I removed this change.